Dockerfile

```
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM ghcr.io/cybozu/golang:1.23-jammy as builder
+FROM ghcr.io/cybozu/golang:1.24-jammy as builder
```

```suggestion
FROM ghcr.io/cybozu/golang:1.24-jammy AS builder
```

Would you make `as` upper case?

---

Thank you!